### ISSUE TITLE
Error 6001 Again

### ISSUE THREAD

**Issue by BunniMagi** [ShikyoKira/Project-New-Reign---Nemesis-Main#367]
Hello I'm for some reason getting this error

**Initializing engine update

ERROR(6001): An exception has occured while updating. Report to Nemesis' author immediately
Message: No mapping for the Unicode character exists in the target multi-byte code page.

Failed to update engine**

I am on a English local with only English as a typeable language and I have no idea what could possibly be causing this unicode error. It only happens if I am using the most up to date

However if i downgrade to the previous version, I then get nemesis claiming i have 32bit hxk files in my skyrim directory despite those files not existing in there at all or my overwrite folder. (i use mo2)

So i honestly haveno idea what to do especially since nemesis was working literally fine 4 hours ago

**Comment by ShikyoKira** [comment]
> However if i downgrade to the previous version, I then get nemesis claiming i have 32bit hxk files in my skyrim directory despite those files not existing in there at all or my overwrite folder. (i use mo2)

look harder. It won't report something that doesn't exist. because you are using mo2, mo2 redirects those files to skyrim directory, so look for them in your mod list

Test it in a vanilla setup unless it has to do with other mods to reproduce the bug. This is a beta tool, so I expect people to do a thorough test before reporting to me their findings

**Comment by BunniMagi** [comment]
I have looked through my modlist and even tried scanning all my folders for the supposed LE animations but nothing came up. After many trial and errors and having to reinstall it constantly I managed to get the newest version of nemesis to work and it didnt report me about the hxk files so I am really unsure of what nemesis was reading as i have literally looked for the specific animations it reported errors on. But in the end, its fixed and working no so I will say this can be closed.
